utils.js

```diff
@@ -11,15 +11,14 @@
  * for the specific language governing rights and limitations under the
  * License.
  *
- * The Original Code is Home Dash Utility.
+ * The Original Code is Vertical Tabs.
  *
- * The Initial Developer of the Original Code is The Mozilla Foundation.
+ * The Initial Developer of the Original Code is
+ * Philipp von Weitershausen.
  * Portions created by the Initial Developer are Copyright (C) 2011
  * the Initial Developer. All Rights Reserved.
  *
  * Contributor(s):
- *   Edward Lee <edilee@mozilla.com>
- *   Erik Vold <erikvvold@gmail.com>
```

Comment:
Those contributors were for the Home Dash Utility (unload / watchers) which no longer exist. This utils file was original a stripped down version of https://github.com/mozilla/prospector/blob/master/homeDash/scripts/utils.js

The real reason is actually that I just deleted this file as a part of my refactor and then later remade it and copy pasted a header :P. I can revert this if you want.